src/box/index_def.h

```diff
@@ -404,7 +404,7 @@ index_def_cmp(const struct index_def *key1, const struct index_def *key2);
  * @param old_space   space definition
  */
 bool
-index_def_is_valid(struct index_def *index_def, const char *space_name);
+index_def_check_is_valid(struct index_def *index_def, const char *space_name);
```

Comment:
Nit: Conventionally, a function that sets a diag should return 0 on success, -1 on error. Please fix and update the comment.

Also, why not name it simply `index_def_check`? Note, we already have `index_def_check_xc`, which invokes this function and throws on error.

Reply:
Thank you for the review! Fixed.